includes/options/class-amp-options-menu.php

```diff
@@ -573,4 +574,47 @@ public function render_screen() {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Possibly replace the message of the 'Settings saved' notice.
+	 *
+	 * If this is the first time a user has changed a setting here,
+	 * display 'View your site as AMP...' instead of 'Settings saved'.
+	 * Altering a setting in the global $wp_settings_errors is not ideal.
+	 * But it looks like there is no filter for this.
+	 */
+	public function possibly_replace_settings_saved_notice() {
+		global $wp_settings_errors;
+		$meta_key = 'amp_view_your_site_notice';
+		$user_id  = get_current_user_id();
+
+		// Exit if this setting isn't present, or if this notice has already displayed.
+		if ( ! get_settings_errors( 'general' ) || get_user_meta( $user_id, $meta_key, true ) ) {
+			return;
+		}
+
+		foreach ( $wp_settings_errors as $key => $setting ) {
+			if ( 'general' === $setting['setting'] && 'settings_updated' === $setting['code'] ) {
+				switch ( AMP_Options_Manager::get_option( 'theme_support' ) ) {
+					case 'native':
+						$message = esc_html__( 'Native Mode activated! View your site as AMP now or Review Errors', 'amp' );
+						break;
+					case 'paired':
+						$message = esc_html__( 'Paired Mode activated! View your site as AMP now or Review Errors', 'amp' );
+						break;
+					case 'classic':
+						$message = esc_html__( 'Classic Mode activated! View your site as AMP now. We recommend upgrading to Native or Paired mode.', 'amp' );
+						break;
+				}
+
+				if ( isset( $message ) ) {
+					$wp_settings_errors[ $key ]['message'] = $message; // WPCS: Global override OK.
+					// Ensure that this notice does not display again.
+					update_user_meta( get_current_user_id(), $meta_key, true );
```

Comment:
We can remove the user meta flag. The notice should be shown each time the user changes the mode.

Reply:
Good point. https://github.com/Automattic/amp-wp/pull/1443/commits/3a752500c5573e53217d4abfb33ea7172b88b82c removes the user meta as a condition of displaying this message.